### ISSUE TITLE
What about Ubuntu 18.04?

### ISSUE THREAD

**Issue by ViBE-HU** [andrebrait/prime-indicator#33]
Maybe it's a lame question. I guess there are already a solution for this on system level (nvidia-prime) but on my freshly installed 18.04 beta it's absolutely dead. Do you have any plans on the new OS?

**Comment by andrebrait** [comment]
I promise I'll take a look into those things as soon as I graduate (probably July 2018). ;)

**Comment by mmhobi7** [comment]
instead of a log out it needs to be rebooted, and the nvidia power stuff doesn't work anymore

**Comment by andrebrait** [comment]
@Aaahh Yes, I just tested some time ago. As a matter of fact, it's funky even on 16.04.3 and newer.

**Comment by Motherboard** [comment]
@Aaahh It also happens in 17.10, I suspect it's because of GDM - I've installed Cinnamon and LightDM instead of Gnome and GDM on my 17.10, and now it works like a charm - I guess it's the same thing with regards to 18.04.

**Comment by mmhobi7** [comment]
@Motherboard 
I have Lightdm and it's still funky

**Comment by jasmines** [comment]
I'm trying to contact the developer since ubuntu 17.10 asking for an upgrade.
Never answered.
The ppa is active but prime-indicator-plus package must have been abandoned...

Not a great behaviour, but that's it...

**Comment by andrebrait** [comment]
@jasmines I haven't released a new version of this software, which is why the guy responsible for the PPA hasn't updated it, AFAIK. 

If by "the developer" you mean me, though, I'm sorry that I didn't respond, but I wasn't notified of anyone trying to contact me as well :/

**Comment by jasmines** [comment]
So what should one do to have a build for bionic in the PPA? Can't you put us in contact with the mantainer? He should be the guy behind webupd8 and he's ignoring our messages as well.

**Comment by andrebrait** [comment]
@jasmines I had a discussion that included him earlier this year and he didn't respond either.

I don't know what happened to him. I hope he's okay. I'll see if he can respond the email.

Bear in mind that the indicator isn't exactly in the best shape for bionic, as others have pointed out... :/

**Comment by mmhobi7** [comment]
Git clone and run the install script

**Comment by andrebrait** [comment]
@Aaahh yeah, that works too.

And installing the dependencies like in the README for this repo too.

**Comment by ViBE-HU** [comment]
@jasmines sadly [Andrei](https://plus.google.com/+AlinAndrei) (the maintainer of that PPA and the creator of WebUpd8) is pretty inactive for a while so it's not gonna be easy.

**Comment by jasmines** [comment]
A while means more than a year for the PPA. Webupd8 is getting updated regularly, instead, as it gives him money...Il 24 mag 2018 7:26 AM, ViBE <notifications@github.com> ha scritto:@jasmines sadly Andrei (the maintainer of that PPA and the creator of WebUpd8) is pretty inactive for a while so it's not gonna be easy.

**Comment by Motherboard** [comment]
[This](https://github.com/matthieugras/Prime-Ubuntu-18.04) might be of interest - a replacement for the prime-select in Ubuntu 18.04 which works without reboot.

**Comment by andrebrait** [comment]
Thanks for the tip! I'll look into it.

On top of it all, it's written in Rust :) I like Rust

**Comment by Motherboard** [comment]
That nvidia-prime alternative I posted seems to be not maintained at the moment, it had been forked and further improved [here](https://github.com/matthieugras/Prime-Ubuntu-18.04), there's an active discussion about it in the [nvidia dev forum](https://devtalk.nvidia.com/default/topic/1032482/linux/optimus-on-ubuntu-18-04-is-a-step-backwards-but-i-found-the-first-good-solution/4).

**Comment by jasmines** [comment]
And the PPA is still missing updates... Launchpad should be use to close them when the mantainer even refuse to answer...
